README.md

```diff
@@ -0,0 +1,195 @@
+
+* 📥 [**Install Rudder**](https://docs.rudder.io/reference/current/installation/index.html), with or without a subscription.
+* You can then follow the [get started documentation](https://docs.rudder.io/get-started/current/configuration-policies/index.html) to discover the features with a hands-on tutorial.
+
+### Quick installation
+
+Alternatively, for a quick setup on Linux systems, you can use the installer script.
+For a Rudder server:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://repository.rudder.io/tools/rudder-setup | sh -s -- setup-server latest
+```
+
+For a Rudder agent, replace `SERVER` by you Rudder server's IP or hostname:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://repository.rudder.io/tools/rudder-setup | sh -s -- setup-agent latest SERVER
+```
+
+For more options and to review the script before running it:
+
+```bash
+wget https://repository.rudder.io/tools/rudder-setup
+chmod +x rudder-setup
+./rudder-setup -h
+```
+
+## FAQ
+
+
+### Is Rudder open-source? Is it free? What is Rudder Core?
+
+Rudder has adopted an _open-core_ model:
+
+* **Rudder Core** is available for free and totally open-source (under GPL 3.0 an Apache 2.0). It includes the main Rudder components, Linux agents, plus several plugins.
+* The complete **Rudder** solution is paid and partially open-source. A part of the paid plugins and agents are distributed under proprietary license. It also includes other benefits:
+
+- Additional plugins that add major features
+- Support for older versions on Linux distributions
+- Support for additional architectures (ARM & Power)
+- Support for proprietary operating systems: Windows and AIX
+- Extended support for Rudder releases
+- Professional support, user assistance
+- Training
+
+They serve different purposes:
+
+* **Rudder Core** is a versatile automation engine for mainstream Linux systems, built to give access to essential configuration management and inventory features to small businesses and individuals.
+* **Rudder** is a multi-platform operational security solution, answering the needs of most organizations, including enterprise setup and critical infrastructures.
+
+Rudder Core is a subset of the full solution, and the packages publicly available are identical the packages provided to subscription users, with the same QA and security processes. This is not a two-speed model, but a "two-scope" model.
+
+### Do I need to reinstall anything if upgrading from Rudder Core to a subscription?
+
+The full Rudder solution is a direct super-set of Rudder Core, so you don't have to
+reinstall anything and your server and agents are compatible with the additional
+features which come as plugins.
+
+You may just need to change the repository URL on your systems to benefit from the extended
+maintenance in the future.
+
+### What is the governance structure of the Rudder project?
+
+The project is lead by the French (🇫🇷🇪🇺) company of the same name (previously known as *Normation*).
```

Comment:
"French laws apply to the project."
Or something to give the legal juridiction

Reply:
(it's implied by french company, perhaps it's enough actually)

Reply:
I think the country is enough, especially in a readme.

Speaking about legal jusrisdiction looks a bit threatening to me.